plugins/community/neoeinstein-prost/v0.2.3/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/community/neoeinstein-prost
+plugin_version: v0.2.3
+source_url: https://github.com/neoeinstein/protoc-gen-prost
+description: Generates code using the Prost! code generation engine.
+output_languages:
+  - rust
+spdx_license_id: Apache-2.0
+license_url: https://github.com/neoeinstein/protoc-gen-prost/blob/1a6d3593622af18b75a4a79f545f8530cdaf444f/LICENSE
```

Comment:
Noticed the link to the tag wasn't working so using this for now - opened https://github.com/neoeinstein/protoc-gen-prost/issues/74 upstream to see if we should use these going forward.